index.css

```diff
@@ -1,28 +1,29 @@
-:root {
-	--max-content-width: 70em;
-	--vis-color-blue: #07508E;
-	--vis-color-yellow: #FFAF12;
-}
-
 html, body {
-	margin: 0;
-	padding: 0;
+	margin: 0em;
+	padding: 0em;
 	font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+	text-align: center;
 }
 
 a {
 	text-decoration: underline;
 	color: inherit;
 }
 
+.content {
+	width: 100%;
+	max-width: 70em;
+	margin: 0px auto;
+	padding: 0px;
+}
+
 header {
 	padding: 1em;
-	text-align: center;
 
 	color: white;
 	text-shadow: 1px 1px 3px rgba(0, 0, 0, 1);
 
-	background: var(--vis-color-blue)9; /* Old browsers */
+	background: #07508E; /* Old browsers */
```

Comment:
Pardon me, but I wonder why did you remove the CSS variables part. Using CSS Variables makes the color scheme of website more flexible.

Reply:
And absent in IE 11.